src/lib/Scenes/MyCollection/Screens/Artwork/Components/MyCollectionArtworkHeader.tsx

```diff
@@ -24,6 +24,55 @@ const MyCollectionArtworkHeader: React.FC<MyCollectionArtworkHeaderProps> = (pro
 
   const isImage = (toCheck: any): toCheck is Image => !!toCheck
 
+  const imageIsProcessing = (image: Image | null) => {
+    if (!image) {
+      return false
+    }
+
+    const isProcessing = image.height === null
+    return isProcessing
+  }
+
+  const hasImagesStillProcessing = (mainImage: any, imagesToCheck: MyCollectionArtworkHeader_artwork["images"]) => {
+    if (!isImage(mainImage) || imageIsProcessing(mainImage)) {
+      return true
+    }
+
+    if (!imagesToCheck) {
+      return false
+    }
+
+    const concreteImages = imagesToCheck as Image[]
+    const stillProcessing = concreteImages.filter((image) => imageIsProcessing(image)).length > 0
```

Comment:
nonblocking: js has the `some(fn)` method which is the same thing as `.filter(fn).length > 1` but with better perf

```suggestion
    const stillProcessing = concreteImages.some((image) => imageIsProcessing(image))
```

Reply:
Nice